services/QuillLMS/app/views/pages/translations.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Translations</h1>
+<table class="table">
+  <tr class="header-row">
+    <th>Module</th>
+    <th>English</th>
+    <th>Spanish</th>
+  </tr>
+    <%@translations.each do |translation|%>
+      <tr class="data-row">
+        <td><%= translation.source.class.name %></td>
```

Comment:
This is not a new problem in your code, but you've probably noticed that we often have Model names that don't match what Curriculum calls something.  `ConceptFeedback` vs "Hint" for instance.  Not a problem with this early version, but I wonder if it makes sense to consider including a way to show what Curriculum calls these values somewhere instead of making them learn the underlying model names.

I think solving this issue is outside the scope of this PR, though, so not something you need to solve here.

Reply:
FWIW concept feedback is referred to that way in other parts of the admin UI, so they do know what that means in this instance. If we do need to use something different for display at a later date, I think we could just create a variable to store a key value pair for this.